docs/cli/cmd-options.mdx

```diff
@@ -113,113 +113,7 @@ keywords:
   - workflow visibility
   - x509-certificate
 tags:
-  - actions
-  - active-cluster
-  - activity
-  - activity-execution
-  - activity-id
-  - address
-  - archival
-  - backfill
-  - batch-job
-  - build
-  - build-id
-  - ca-certificate
-  - calendar
-  - certificate-key
-  - child-workflows
-  - cli-reference
-  - cluster
-  - codec-server
-  - command-line-interface-cli
-  - command-line-interface-cli,
-  - concurrency-control
-  - configuration
-  - context
-  - continue-as-new
-  - cron
-  - cross-cluster-connection
-  - data-converters
-  - endpoint
-  - environment
-  - event
-  - event-id
-  - event-type
-  - events
-  - external-temporal-and-state-events
-  - failures
-  - frontend
-  - frontend-address
-  - frontend-service
-  - goroutine
-  - grpc
-  - history
-  - http
-  - interval
-  - ip-address
-  - job-id
-  - log-feature
-  - logging
-  - logging-and-metrics
-  - memo
-  - metrics
-  - namespace
-  - namespace-description
-  - namespace-id
-  - namespace-management
-  - nondeterministic
-  - notes
-  - operation
-  - operator
-  - options-feature
-  - overlap-policy
-  - pager
-  - port
-  - pragma
-  - queries-feature
-  - query
-  - requests
-  - reset-point
-  - resets-feature
-  - retention-policy
-  - retries
-  - reuse-policy
-  - schedule
-  - schedule-backfill
-  - schedule-id
-  - schedule-pause
-  - schedule-unpause
-  - schedules
-  - search-attribute
-  - search-attributes
-  - server
-  - server-options-and-configurations
-  - sqlite
-  - start-to-close
-  - task-queue
-  - task-queue-type
-  - temporal-cli
-  - temporal-ui
-  - time
-  - time-zone
-  - timeout
-  - timeouts-and-heartbeats
-  - tls
-  - tls-server
-  - uri
-  - verification
-  - visibility
-  - web-ui
-  - workflow
-  - workflow-execution
-  - workflow-id
-  - workflow-run
-  - workflow-state
-  - workflow-task
-  - workflow-task-failure
-  - workflow-type
-  - workflow-visibility
-  - x509-certificate
+  - Temporal CLI
```

Comment:
Looks great! There were a lot of this page - definitely made the page look a bit cluttered at the bottom.